From: Andrew Morton <akpm@linux-foundation.org>
Subject: swap-stop-add-to-avail-list-is-swap-is-full-checkpatch-fixes
Date: Tue Jun 27 12:28:53 PM PDT 2023

fix typo in comment

WARNING: 'avaliable' may be misspelled - perhaps 'available'?
#60: FILE: mm/swapfile.c:2332:
+	/* add to avaliable list iff swap device is not full */
 	          ^^^^^^^^^

total: 0 errors, 1 warnings, 11 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

./patches/swap-stop-add-to-avail-list-is-swap-is-full.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Hugh Dickins <hughd@google.com>
Cc: Ma Wupeng <mawupeng1@huawei.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/swapfile.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/swapfile.c~swap-stop-add-to-avail-list-is-swap-is-full-checkpatch-fixes
+++ a/mm/swapfile.c
@@ -2329,7 +2329,7 @@ static void _enable_swap_info(struct swa
 	 */
 	plist_add(&p->list, &swap_active_head);
 
-	/* add to avaliable list iff swap device is not full */
+	/* add to available list iff swap device is not full */
 	if (p->highest_bit)
 		add_to_avail_list(p);
 }
_
